wolfcrypt/src/random.c

```diff
@@ -2455,7 +2455,33 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
     }
 
 #elif defined(WOLFSSL_ESPIDF)
+
+    /* Espressif */
```

Comment:
No, I'd prefer the code blocks to be the same (since they are identical). I was just wondering if there was a more generic ESP32 macro we should support here like `WOLFSSL_ESP32_SYSRNG`?

Reply:
aha! There _is_ an [ESP32 hardware random number generator](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/random.html#random-number-generation). 

Do you think it should be optional for ESP32 or change the code to always use it?

Reply:
I would think always use it. If a user wants to customize they can use the `CUSTOM_RAND_GENERATE`.

Reply:
agreed. perhaps beyond the scope of this PR for ESP8266 support; What do you think of leaving the duplicate code between ESP32 and ESP8266, and create a separate PR to use hardware RNG for the ESP32?

Reply:
Yes that sounds good